src/cromshell/utilities/io_utils.py

```diff
@@ -1,44 +1,148 @@
-    print(colored(r"\   _-'                                                                `-_   /", "red"))
-    print(colored(r".`''                                                                      ``'.", "red"))
+
+    print(
+        colored(
+            r"""
+   =================     ===============     ===============   ========  ========
+   \\ . . . . . . .\\   //. . . . . . .\\   //. . . . . . .\\  \\. . .\\// . . //
+   ||. . ._____. . .|| ||. . ._________________________. . .|| || . . .\/ . . .||
+   || . .||   ||. . || || . ./|  ,,    ,,     ,,      |\. . || ||. . . . . . . ||
+   ||. . ||   || . .|| ||. . ||  \‾\,-/‾/====/‾/,     || . .|| || . | . . . . .||
+   || . .||   ||. _-|| ||-_ .||   \/=<‾><‾><‾><‾>-,   ||. _-|| ||-_.|\ . . . . ||
+   ||. . ||   ||-'  || ||  `-||   / (\‾\‾|‾/‾/‾/‾     ||-'  || ||  `|\_ . .|. .||
+   || . _||   ||    || ||    ||  \‾x/ ˙'-;-;-'˙       ||    || ||   |\ `-_/| . ||
+   ||_-' ||  .|/    || ||    \|_______________________|/    || ||   | \  / |-_.||
+   ||    ||_-'      || ||      `-_||    || ||    ||_-'      || ||   | \  / |  `||
+   ||    `'         || ||         `'    || ||    `'         || ||   | \  / |   ||
+   ||            .===' `===.         .==='.`===.         .===' /==. |  \/  |   ||
+   ||         .=='   \_|-_ `===. .==='   _|_   `===. .===' _-|/   `==  \/  |   ||
+   ||      .=='    _-'    `-_  `='    _-'   `-_    `='  _-'   `-_  /|  \/  |   ||
+   ||   .=='    _-'          `-__\._-'         `-_./__-'         `' |. /|  |   ||
+   ||.=='    _-'                                                     `' |  /==.||
+   =='    _-'                                                            \/   `==
+   \   _-'                                                                `-_   /
+   .`''                                                                      ``'.
+            """,
+            "red",
+        )
+    )
+
+
+def assert_file_is_not_empty(file_path: str or Path, file_description: str):
+    """Confirm the provided file exist and is not empty."""
+
+    if not Path(file_path).exists():
+        LOGGER.error("ERROR: %s does not exist: %s", file_description, file_path)
+        raise FileExistsError(f"ERROR: {file_description} does not exist: {file_path}")
+    if os.stat(file_path).st_size == 0:
+        LOGGER.error("ERROR: %s is empty: %s.", file_description, file_path)
+        raise EOFError(f"ERROR: {file_description} is empty: {file_path}.")
+
+
+def is_workflow_id_valid(workflow_id: str):
+    """Validates a workflow id"""
+
+    if workflow_id == "":
+        raise ValueError("Empty String")
+
+    return workflow_id_pattern.match(workflow_id)
+
+
+def pretty_print_json(json_text: str):
+    """Prints JSON String in a fancy way
+
+    - json_text: content of the json, NOT json file path"""
+
+    loaded_json = json.loads(json_text)
+    print(json.dumps(loaded_json, indent=4, sort_keys=True))
```

Comment:
Instead of printing directly to stdout it's usually good practice to oeither output to a String or accept a stream to print to as an input.  This makes testing awkward and is less flexible to use.